tsconfig.base.json

```diff
@@ -12,8 +12,6 @@
         "module": "amd",
         "moduleResolution": "node",
         "noFallthroughCasesInSwitch": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
```

Comment:
This has been bugging me for a while. TypeScript `-b` mode won't create outputs when there are errors, even from unused locals/parameters.